lib/src/entities.ts

```diff
@@ -34,6 +39,23 @@ export interface FormValidationFunction {
   (vm: any): ValidationResult;
 }
 
-export interface ValidationConstraints extends Object {
+export interface SyncValidationFunction {
+  (value: any, vm: any, customParams: any): ValidationResult;
+}
+
+export interface AsyncFieldValidationFunction {
```

Comment:
Why do we need defined `AsyncFieldValidationFunction` if we define interface like `ValidationResult`?